test/function/samples/empty-string-as-module-name/_config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
By default, a `function` test `_config.js` file only needs to contain a `description` line which is added to the test output. If nothing else is specified, the test will assume there is a file named `main.js`. It will attempt to bundle that file and run it.